infra/experimental/SystemSan/SystemSan.cpp

```diff
@@ -76,6 +76,20 @@ constexpr int kRootDirMaxLength = 16;
 
 // The PID of the root process we're fuzzing.
 pid_t g_root_pid;
+
+// Structure to know which thread in the root process triggered the bug.
+struct ThreadParent {
+  // PID/TID of parent
+  pid_t pid;
+  // Root process
+  bool root = true;
+
+  ThreadParent() : pid(0) {};
```

Comment:
nit: I reckon the `;` at is not necessary? And it breaks the consistency with the next line : )

Reply:
+1

Reply:
Ok